lib/runners/processor/cppcheck.rb

```diff
@@ -67,6 +70,10 @@ def language
       config_linter[:language].then { |lang| lang ? ["--language=#{lang}"] : [] }
     end
 
+    def jobs
+      (config_linter[:parallel] && !config_linter[:project]) ? ["-j", Etc.nprocessors.to_s] : []
```

Comment:
[suggestion] How about adding a warning as follows?
(Can we give a good reason for the limitation to users?)

```suggestion
      @jobs ||=
        if config_linter[:parallel]
          if config_linter[:project]
            add_warning <<~MSG, file: config.path_name
              The `parallel` option is ignored when the `project` option is specified.
              This limitation is due to the behavior of #{analyzer_name}.
              See also <https://...>
            MSG
            []
          else
            ["-j", Etc.nprocessors.to_s]
          end
        else
          []
        end
      end
```

Reply:
I tried to search the Web but could not find any notes, reports, or issues about this limitation. Shall we omit the "See also" line as of now?

Reply:
Yes, of course. 👍

Reply:
Fixed. commit 5210f88bb69a19b514f3386dd96122a108149622